bbc_stock/migrations/8.0.1.1/pre-migrate.py

```diff
@@ -0,0 +1,7 @@
+# coding: utf-8
+def migrate(cr, version):
+    if not version:
+        return
+    cr.execute(
+        """ DELETE FROM ir_translation
```

Comment:
If you only *add* a translation you don't have to clear existing translations.

Reply:
@StefanRijnhart  You are right. Solved in my last commit.